Improve the Windows Agent docs (#24543)
DataDog/documentation: content/en/agent/basic_agent_usage/windows.md

- Name the group (e.g., `DatadogAgentsGroup`), set the correct group scope for your organization (e.g. **Domain local**), and the type to **Security**.

**2. Create the gMSA:**

- Open PowerShell with **Administrator** privileges.
- Run the following command to create the gMSA:
```powershell
New-ADServiceAccount -Name DatadogGMSA -DNSHostName <YOUR_DOMAIN_NAME> -PrincipalsAllowedToRetrieveManagedPassword DatadogAgentsGroup
```

Replace `<YOUR_DOMAIN_NAME>` with your domain name.

**3. Install the gMSA on the Target Machine:**

- Ensure the target machine is part of the `DatadogAgentsGroup`.
- On the target machine, open PowerShell and run:
```powerhsell
Install-ADServiceAccount -Identity DatadogGMSA
```
### Install the Datadog Agent
Download the [Datadog Agent installer][1] to install the latest version of the Agent.

   <div class="alert alert-info">If you need to install a specific version of the Agent, see the <a href="https://ddagent-windows-stable.s3.amazonaws.com/installers_v2.json">installer list</a>.</div>

#### Install via the GUI
- Run the installer by opening `datadog-agent-7-latest.amd64.msi`. When prompted, enter your Administrator credentials.

> - Follow the prompts, accept the license agreement, and enter your [Datadog API key][2].

Reviewer: ```suggestion
1. Follow the prompts, accept the license agreement, and enter your [Datadog API key][2].
```

> - When prompted for the "Datadog Agent User Account", enter the username of the gMSA (e.g., `<YOUR_DOMAIN_NAME>\DatadogGMSA$`) and **no password**.

Reviewer: ```suggestion
1. When prompted for the "Datadog Agent User Account", enter the username of the gMSA. For example, `<YOUR_DOMAIN_NAME>\DatadogGMSA$`) and **no password**.
```

> - When the install finishes, you are given the option to launch the Datadog Agent Manager.

Reviewer: ```suggestion
When the install finishes, you are given the option to launch the Datadog Agent Manager.
```

#### Install with the command line

> - Open PowerShell with **Administrator** privileges.
> - Run the following command to install the Datadog Agent:

Reviewer: ```suggestion
1. Open PowerShell with **Administrator** privileges.
1. Run the following command to install the Datadog Agent:
   **Note:** Replace `DatadogGMSA$` with the username of your gMSA. The username **must end with a $ symbol.**
```

> ```powershell

Reviewer: ~~~suggestion
   ```powershell
~~~

Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-agent-7-latest.amd64.msi APIKEY="<YOUR_DATADOG_API_KEY>" DDAGENTUSER_NAME="<YOUR_DOMAIN_NAME>\DatadogGMSA$'

> ```

Reviewer: ~~~suggestion
   Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-agent-7-latest.amd64.msi APIKEY="<YOUR_DATADOG_API_KEY>" DDAGENTUSER_NAME="<YOUR_DOMAIN_NAME>\DatadogGMSA$'
   ```
~~~

> **note:** Replace `DatadogGMSA$` with the username of your gMSA, it **must end with a $ symbol.**

Reviewer: ```suggestion
```

[1]: https://app.datadoghq.com/account/settings/agent/latest?platform=windows
[2]: /agent/supported_platforms/?tab=windows
[3]: /agent/faq/windows-agent-ddagent-user/
[4]: https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/group-managed-service-accounts/group-managed-service-accounts/group-managed-service-accounts-overview#software-requirements
[5]: https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/group-managed-service-accounts/group-managed-service-accounts/getting-started-with-group-managed-service-accounts

{{% /tab %}}
{{< /tabs >}}

> ##### Installation Configuration Options 

Reviewer: ```suggestion
##### Installation configuration options 
```

| `PROCESS_ENABLED`                           | String  | Enable (`"true"`) or disable (`"false"`) the Process Agent in the configuration file. The Process Agent is disabled by default.                                                                                                     |
| `HOSTNAME_FQDN_ENABLED`                     | String  | Enable (`"true"`) or disable (`"false"`) the usage of FQDN for the Agent hostname. It is equivalent to set `hostname_fqdn` in the Agent configuration file. The usage of FQDN for the hostname is disabled by default. _(v6.20.0+)_ |
| `CMD_PORT`                                  | Number  | A valid port number between 0 and 65534. The Datadog Agent exposes a command API on port 5001. If that port is already in use by another program, the default may be overridden here.                                               |

> | `PROXY_HOST`                                | String  | If using a proxy, sets your proxy host. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
> | `PROXY_PORT`                                | Number  | If using a proxy, sets your proxy port. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
> | `PROXY_USER`                                | String  | If using a proxy, sets your proxy user. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
> | `PROXY_PASSWORD`                            | String  | If using a proxy, sets your proxy password. For the process/container Agent, this variable is required for passing in an authentication password and cannot be renamed. [Learn more about using a proxy with the Datadog Agent][4]. |

Reviewer: ```suggestion
| `PROXY_HOST`                                | String  | (If using a proxy) sets your proxy host. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
| `PROXY_PORT`                                | Number  | (If using a proxy) sets your proxy port. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
| `PROXY_USER`                                | String  | (If using a proxy) sets your proxy user. [Learn more about using a proxy with the Datadog Agent][4].                                                                                                                                 |
| `PROXY_PASSWORD`                            | String  | (If using a proxy) sets your proxy password. For the process/container Agent, this variable is required for passing in an authentication password and cannot be renamed. [Learn more about using a proxy with the Datadog Agent][4]. |
```

| `EC2_USE_WINDOWS_PREFIX_DETECTION`          | Boolean | Use the EC2 instance id for Windows hosts on EC2. _(v7.28.0+)_                                                                                                                                                                      |

> | [DEPRECATED] `ADDLOCAL` | String | Enable additional agent component. Setting to `"MainApplication,NPM"` causes the driver component for [Network Performance Monitoring][5] to be installed. _(version 7.44.0 and previous)_ |

Reviewer: ```suggestion
| [DEPRECATED] `ADDLOCAL` | String | Enable additional Agent component. Setting to `"MainApplication,NPM"` causes the driver component for [Network Performance Monitoring][5] to be installed. _(version 7.44.0 and previous)_ |
```
